### ISSUE TITLE
Clarify difference between .model and .bind()

### ISSUE THREAD

**Issue by ebidel** [Polymer/docs#116]
TL;DR; `.model` is declarative sugar.

Setting `.model` and using the bind attribute is declarative sugar. When you use
`<template bind="{{ salutations }}">`, MDV takes care of all the `bind()` calls for you under the hood when you use `<template bind="{{ salutations }}">`. 

Simple example: 

    <div id="#div">{{what}}</div>
    <script>
      var model = { salutations: {what: 'GoodBye', who: 'Imperative' , list: [1,2,3]}};
      document.getElementById('#div').firstChild.bind('textContent', model, 'salutations.what');
    </script>

This binds the value in model.what to div.textContent..and this is only one binding.
With a complex template, things get quickly out of hand:

    <template id="greeting" bind="{{ salutations }}">
        Hello, {{who}} - {{what}}
        <p>{{who}} <span class="{{class: who}}">{{what}}</span></p>
        <ul>
          <template repeat="{{l in list}}">
            <li><input type="text" value="{{l}}> out of {{list.length}}</li>
          <template>
        </ul>
    </template>
    <script>
      document.getElementById('#greeting').model = model;
    </script>

That's 9+ binds? MDV is scoping/resolving paths and setting up all the bindings on the different types of nodes.

**Comment by ebidel** [comment]
This has gone stale. Marking as obsolete.
